frontend/src/metabase/modes/components/drill/ActionClickDrill/utils.ts

```diff
@@ -58,25 +50,13 @@ function isMappedParameter(
 
 export function getNotProvidedActionParameters(
   action: WritebackAction,
-  parameterMappings: ActionParametersMapping[],
```

Comment:
This function is basically doing a diff between

1. all the parameters the action needs and
2. the parameter values already provided by the dashboard for this action

If we pass this function the correct data for (2), we don't need to look at the mappings again, so we can eliminate one of the arguments for this function and simplify some of its logic.